supreme/src/commonMain/kotlin/at/asitplus/signum/supreme/validate/BasicConstraintsValidator.kt

```diff
@@ -0,0 +1,42 @@
+package at.asitplus.signum.supreme.validate
+
+import at.asitplus.signum.BasicConstraintsException
+import at.asitplus.signum.indispensable.pki.X509Certificate
+import at.asitplus.signum.indispensable.pki.pkiExtensions.BasicConstraintsExtension
+
+class BasicConstraintsValidator(
```

Comment:
Could the validators which don't depend on any crypto APIs be moved to the indispensable module, so one can combine the certificate code with a different crypto API (e.g. cryptography-kotlin)?